### PR TITLE
Create make_tree_from_inorder_and_postorder.c

### PR DESCRIPTION
**Pull request by ervaibhavkumar**
**Changes:**
Added Code to make tree from in order and post order traversal in C language

### CODE REVIEW

File: code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	int data;
+	struct btree *left;
+	struct btree *right;
+}btree;
+
+btree *makeTree(int *in , int *post , int start , int end , int *index) {
+	if (start > end) return NULL;
+
+	btree *nn = (btree*)malloc(sizeof(btree));
+	nn->data = post[*index];
```
Comment (sibasish14) [inline code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c]: Space between `->`

File: code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	int i;
+	for (i=start;i <= end;i++) {
```
Comment (sibasish14) [inline code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c]: Space between `=`

File: code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+		if (post[*index] == in[i]) {
+			break;
+		}
```
Comment (sibasish14) [inline code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c]: No need of brackets.... there's only one statement `break`

File: code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	}
+	(*index)--;
+	nn->right = makeTree(in,post,i+1,end,index);
```
Comment (sibasish14) [inline code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c]: Space between `->` and `+`

File: code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+	nn->left = makeTree(in,post,start,i-1,index);
```
Comment (sibasish14) [inline code/data_structures/src/tree/binary_tree/binary_tree/make_binary_tree/from_inorder_and_postorder/make_tree_from_inorder_and_postorder.c]: Space between -> and +

**Comment by sibasish14** [comment]
@ervaibhavkumar add spaces everywhere there's an operator. And add one space after commas.

**Comment by ervaibhavkumar** [comment]
@sibasish14 Done with the changes :)

**Comment by ervaibhavkumar** [comment]
@sibasish14 Done :+1:

**Comment by sibasish14** [comment]
LGTM 👍

**Comment by ervaibhavkumar** [comment]
@AdiChat Please see this.

**Comment by ervaibhavkumar** [comment]
@arnavb Please see to it as well

**Comment by arnavb** [comment]
djldk 2

**Pull request merged**